ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
Yotta versioning should prevent sudden failures. 

The original constructor is already being used "both ways" - its the only way to make service discovery and advertisement work at the same time. 

If we are going to have constructors for both, they should be labeled appropriately and the old one should be deprecated since it is too ambiguous, 

```
UUIDMSB(const LongUUIDBytes_t)
UUIDLSB(const LongUUIDBytes_t)
setupLongMSB(const LongUUIDBytes_t)
setupLongLSB(const LongUUIDBytes_t)
```